dallinger/frontend/static/scripts/dallinger2.js

```diff
@@ -28,6 +48,24 @@ var dallinger = (function () {
     }
   };
 
+  /**
+   * ``dallinger.identity`` provides information about the participant.
+   * It has the following string properties:
+   *
+   * ``recruiter``     - Type of recruiter
+   *
+   * ``hitId``         - MTurk HIT Id
```

Comment:
I don't think these terms are generic, though there's probably something equivalent to them in other similar systems (especially those patterned after MTurk in some way). For now I believe it's best to document them as they are currently used rather than try to use generic language for them that may need to change once the system becomes more generic. In the end this is probably a question for @suchow and @vlall, regarding how we should describe the MTurk derived parameters.